tests/tests_pytorch/strategies/test_fsdp.py

```diff
@@ -142,6 +151,37 @@ def _assert_save_equality(trainer, ckpt_path, cls=TestFSDPModel):
             assert torch.equal(ddp_param.float().cpu(), shard_param)
 
 
+def custom_auto_wrap_policy(
```

Comment:
could we move the definition of these to above the test_fsdp_checkpoint_multi_gpus function where they are used?